src/App.js

```diff
@@ -1,113 +1,122 @@
-import React,{useEffect, useState} from 'react';
-import './App.css';
-import Form from './Components/Form';
-import Header from './Components/Header';
-import TodoList from './Components/TodoList';
-import { BrowserRouter, Route, Switch } from "react-router-dom";
-import Login from './Components/Login';
+import React, { useEffect, useState } from "react"
+import "./App.css"
+import Form from "./Components/Form"
+import Header from "./Components/Header"
+import TodoList from "./Components/TodoList"
+import { BrowserRouter, Route, Switch } from "react-router-dom"
+import Login from "./Components/Login"
+import { createMuiTheme, ThemeProvider } from "@material-ui/core"
+import { orange, red } from "@material-ui/core/colors"
+
+const theme = createMuiTheme({
```

Comment:
Can we refactor this file and separate all the components that can be moved to other file?

Reply:
Created Separate pages and included just routing on App.js

Reply:
Theme should be in a seperate file

Reply:
Created a theme.js file